lib/modules/dosomething/dosomething_northstar/dosomething_northstar.module

```diff
@@ -107,41 +148,39 @@ function _dosomething_northstar_build_http_client() {
 
   return $client;
 }
-//@TODO: use this once we can use guzzle.
-// class Northstar {
-
-//   public $client;
-
-//   public function __construct()
-//   {
-
-//     $this->client = $library;
-//     $base_url = NORTHSTAR_URL;
-//     if (getenv('DS_ENVIRONMENT') === 'local') {
-//       $base_url .=  ":" . NORTHSTAR_PORT;
-//     }
-//     $version = NORTHSTAR_VERSION;
-
-//     $library = libraries_load('guzzle');
-//     //@TODO: this throws an error, because the above call doesn't load the guzzle.
-//     $client = new GuzzleHttp\Client(array(
-//       'base_url' => array($base_url . '/{version}/', array('version' => $version)),
-//       'defaults' => array(
-//         'headers' => array(
-//           'X-DS-Application-Id' => NORTHSTAR_APP_ID,
-//           'X-DS-REST-API-Key' => NORTHSTAR_APP_KEY,
-//           'Content-Type' => 'application/json',
-//           'Accept' => 'application/json'
-//           ),
-//         ),
-//     ));
-//     $this->client = $client;
-//   }
-
-//   public function register($user) {
-//     $client->post('users', array(
-//       'body' => json_encode($user),
-//       ));
-//   }
-
-// }
+class Northstar {
+
+  protected $client;
+
+  public function __construct() {
+    $this->client = $library;
+    $base_url = NORTHSTAR_URL;
+    if (getenv('DS_ENVIRONMENT') === 'local') {
+      $base_url .=  ":" . NORTHSTAR_PORT;
+    }
+    $version = NORTHSTAR_VERSION;
+
+    $library = libraries_load('guzzle');
```

Comment:
Need to ensure that `libraries_load` function returns TRUE before calling below.